parlai/chat_service/services/websocket/websocket_manager.py

```diff
@@ -269,14 +269,14 @@ def observe_payload(self, socket_id, payload, quick_replies=None):
         :param socket_id:
             int identifier for agent socket to send message to
         :param payload:
-            (dict) payload to send through the socket. The keys should be:
-                'image': (True/False) whether the message is an image
-                'mime_type': str. Mime type of the message
-                'data': str. base64 encoded content
+            (dict) payload to send through the socket. The mandatory keys are:
+                    'type': (True/False) whether the message is an image
```

Comment:
nit: it appears the values for `type` are no longer `(True/False)`, could you update that as well?